### DISCUSSION TITLE
Any examples of authentication?

### DISCUSSION THREAD

**Discussion by ndarilek** [benallfree/pocketpages#10] [Q&A]
Trying to put together a prototype and am stumbling a bit on authentication. Are there any examples on how to do this?

Pocketbase appears to have methods to export/load the auth store from cookies, but I can't find these anywhere in the PB docs so bunches of what I'm reading are from GH issues/discussions. My assumption is that loading the cookie would be done in a top-level middleware, so conceptually it seems simple, but auth is tricky enough to get right that if anyone has a simple login/signup/middleware combo I can look to for inspiration, I'd very much appreciate it. That, or pointing me to the cookie methods in the PB docs would help as well.

Thanks!

**Comment by benallfree** [comment]
You might be able to help with this, I'm starting to work on it today.

Here's what I know so far:

PB authorization works by JWT. 

In theory, there should be a way, at the beginning of the request, to grab a cookie and set authorization from that https://github.com/pocketbase/pocketbase/discussions/5307#discussioncomment-10941678

**Reply by ndarilek** [reply]
Interesting, let me know if you get a branch up and I'll take a look. Not super familiar with JSVM so I'd need to get up to speed--if this is something you can wrap up in a day or 2 then I might not be able to help but if it takes much longer then I might.

Thanks!

**Reply by benallfree** [reply]
Auth is in v0.12 release.
